src/match_traildb.c

```diff
@@ -1136,6 +1152,11 @@ int parse_args(int argc, char **argv,
           case 'f': *filter = optarg; break;
           case 'w': *window_file = optarg; break;
           case 'o': *format = optarg; break;
+          case '?':
+          case 'h':
+            print_usage(stderr, argv[0]);
+            exit(EX_USAGE);
```

Comment:
May be nitpicking, but I thought `EX_USAGE` is intended for the case when the binary received incorrect arguments